src/components/MediaItem/MediaItemTitle.tsx

```diff
@@ -68,17 +71,26 @@ export default function MediaItemTitle({ i, data, type, parentRef }: Props) {
     window.addEventListener('resize', debouncedRestartAnimation)
     checkWidth()
 
+    getSession().then((session) =>
```

Comment:
This request runs too many times. We're making a request for every single media item every time it's rendered. Makes sense to move it up higher into a parent.